src/plot.jl

```diff
@@ -0,0 +1,33 @@
+@recipe function f(tt::Table; indices = [1,2])
+    if length(indices) == 2
+        x_name, y_name = propertynames(tt)[indices]
+        xguide --> string(x_name)
+        yguide --> string(y_name)
+        return getproperty(tt, x_name), getproperty(tt, y_name)
```

Comment:
For generic Tables.jl, these would be `Tables.getcolumn(tt, x_name), Tables.getcolumn(tt, y_name)`